content/2-running-automation-testing/guide.md

```diff
@@ -0,0 +1,202 @@
+# 2. Running automation test with Kobiton
+
+## 2.1 Prerequisites
```

Comment:
How about `## 2.1 Prepare Kobiton configuration for executing Automation testing`